test/sslapitest.c

```diff
@@ -651,6 +651,121 @@ static int test_no_ems(void)
 }
 #endif
 
+/*
+ * Very focused test to exercise a single case in the server-side state
+ * machine, when the ChangeCipherState message needs to actually change
+ * from one cipher to a different cipher (i.e., not changing from null
+ * encryption to reall encryption).
```

Comment:
Typo: reall
But feel free to fix it when merging as this is certainly a trivial change.

Reply:
Whoops, I seem to have missed this typo fix :(